recipes/bioconductor-biobase/meta.yaml

```diff
@@ -1,23 +1,23 @@
 package:
   name: bioconductor-biobase
-  version: 2.30.0
+  version: 2.32.0
 source:
-  fn: Biobase_2.30.0.tar.gz
-  url: http://bioconductor.org/packages/release/bioc/src/contrib/Biobase_2.30.0.tar.gz
-  md5: 697d3b5504f899c38a5e6e784ca0e33f
+  fn: Biobase_2.32.0.tar.gz
+  url: https://bioarchive.galaxyproject.org/Biobase_2.32.0.tar.gz
+  md5: acc8c6a85fefcdfd14261c3e3b173e21
 build:
-  skip: False
-  number: 0
+  string: r{{CONDA_R}}_{{PKG_BUILDNUM}}
+  number: 2
   rpaths:
     - lib/R/lib/
     - lib/
 requirements:
   build:
     - 'bioconductor-biocgenerics >=0.3.2'
-    - 'r >=2.10'
+    - 'r {{CONDA_R}}'
```

Comment:
We don't need the CONDA_R here. I am pretty sure that the R version is pinned automatically, similar to Python and Perl.